openresty-build-tools/kong-ngx-build

```diff
@@ -13,6 +13,7 @@ PCRE_VER=
 FORCE=0
 OPENRESTY_PATCHES=$OPENRESTY_PATCHES
 KONG_NGINX_MODULE=master
+RESTY_LMDB=master
```

Comment:
We should probably default disable the lua-resty-lmdb dependency until it is formally released.